### PR TITLE
ROX-9115: E2e test runner for non groovy tests

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

Per title, this adds support for running 'non groovy' e2e tests outside of CI.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

Manually ran without error.

### CODE REVIEW

File: tests/e2e/run-e2e-tests.sh [hunk 1]
```diff
@@ -38,7 +43,11 @@ $script -c qa
 $script qa DeploymentTest 'Verify deployment of type Job is deleted once it completes'
 
 # Run the full set of qa-tests-backend/ tests. This is similar to what CI runs
-# for a PR.
+# for *-qa-e2e-tests jobs on a PR.
+$script qa
+
+# Run the full set of 'non groovy' e2e tests. This is similar to what CI runs
+# for *-nongroovy-e2e-tests jobs on a PR.
 $script qa
```
Comment (porridge) [inline tests/e2e/run-e2e-tests.sh:51]: Also, should the list below `E2e flavor:` be updated?

File: tests/e2e/run-e2e-tests.sh [hunk 2]
```diff
@@ -19,10 +20,14 @@ usage() {
...
+environment.
+
 Options:
-  -c - configure the cluster for test but do not run any tests.
-  -d - enable debug log gathering to '${QA_TEST_DEBUG_LOGS}'.
-  -m - override 'make tag' for the version to install.
+  -c - configure the cluster for test but do not run any tests. [qa flavor only]
+  -d - enable debug log gathering to '${QA_TEST_DEBUG_LOGS}'. [qa flavor only]
+  -m - override 'make tag' for the version to install. [qa flavor only]
   -o - choose the cluster variety. defaults to k8s.
```
Comment (porridge) [inline tests/e2e/run-e2e-tests.sh:31]:
```suggestion
  -o - choose the cluster orchestrator. Either k8s or openshift. defaults to k8s.
```

File: tests/e2e/run-e2e-tests.sh (hunk 1 above)
Comment (porridge) [inline tests/e2e/run-e2e-tests.sh:51]:
```suggestion
$script e2e
```

**Review by porridge — APPROVED** [review]

**Review by janisz — APPROVED** [review]

**Review by janisz — APPROVED** [review]

File: tests/e2e/run-e2e-tests.sh (hunk 1 above)
Comment (gavin-stackrox) [reply tests/e2e/run-e2e-tests.sh:51]: I will TAL after I merge #3933

**Pull request merged**